common/bitcointree/musig2_test.go

```diff
@@ -26,110 +27,107 @@ var lifetime = common.RelativeLocktime{Type: common.LocktimeTypeBlock, Value: 14
 var testTxid, _ = chainhash.NewHashFromStr("49f8664acc899be91902f8ade781b7eeb9cbe22bdd9efbc36e56195de21bcd12")
 
 func TestRoundTripSignTree(t *testing.T) {
```

Comment:
The name of the test is misleading, wdyt of `TestBuildAndSignVtxoTree`? Also the file should be renamed, but i'd do that once we refactor the whole common pkg